core/builtin.py

```diff
@@ -513,15 +522,36 @@ def Popd(argv, dir_stack):
 
   try:
     os.chdir(dest_dir)
+    PrintDirStack(dir_stack)
   except OSError as e:
     util.error("popd: %r: %s", dest_dir, os.strerror(e.errno))
     return 1
 
   return 0
 
 
+DIRS_SPEC = _Register('dirs')
+DIRS_SPEC.ShortFlag('-c')
+DIRS_SPEC.ShortFlag('-l')
+DIRS_SPEC.ShortFlag('-p')
+DIRS_SPEC.ShortFlag('-v')
+
 def Dirs(argv, dir_stack):
-  print(dir_stack)
+  arg, i = DIRS_SPEC.Parse(argv)
+  if arg.l:
+    util.warn('*** dirs -l not implemented ***')
+  # Following `bash` behavior for order of operations
+  if arg.c:
+    del dir_stack[:]
+  elif arg.v:
+    for i, entry in enumerate(dir_stack):
+      print('%2d %s' % (i, entry)) #print(' ' + str(i) + ' ' + entry)
```

Comment:
remove the comment here

Reply:
Sorry, should have caught that before pushing. I'll remove it